src/llmcompressor/transformers/sparsification/sparse_model.py

```diff
@@ -78,23 +75,17 @@ def skip(*args, **kwargs):
         # instantiate compressor from model config
         compressor = ModelCompressor.from_pretrained(pretrained_model_name_or_path)
 
-        # temporarily set the log level to error, to ignore printing out long missing
-        # and unexpected key error messages (these are EXPECTED for quantized models)
-        logger = logging.getLogger("transformers.modeling_utils")
-        restore_log_level = logger.getEffectiveLevel()
-        logger.setLevel(level=logging.ERROR)
```

Comment:
Are we going to see these key error messages from transformers now? It would be nice to avoid this IMO

Reply:
Yes removing this causes the giant key error message to print out. Agreed we should avoid it, @markurtz I added this back in (just renamed logger to transformers_logger) and confirms its working with the new framework

Reply:
We need to figure out a better way around this, as the solution put in place is extremely hacky and fragile. We'll update this later, though, as we get to auditing the transformers pipeline pathways